### PR TITLE
Update for Cycle Unified

### PR DESCRIPTION
**Pull request by abaco**
Just a straightforward port to Cycle Unified. It still uses RxJS, and the Electron version is unchanged.

I haven't used it yet, but the tests are passing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,12 +17,13 @@
     "electron"
   ],
   "peerDependencies": {
-    "@cycle/core": "^6.0.0",
+    "@cycle/run": "^3.1.0",
     "electron-prebuilt": "^0.36.0",
-    "rx": "^4.0.0"
+    "rx": "^5.4.0"
```
Comment (apoco) [inline package.json]: Should this also be `rxjs` to match?

**Review by apoco — COMMENTED** [review]
Could you explain the motivation for adapting all of the RxJS observables to xstreams? I've left this project dormant for quite a while and hadn't heard of them before. This looks like a lightweight alternative to RxJS, but does it have specific advantages beyond it being written by the same developer?

I'm happy if you're able to pick up development on this project, by the way. I didn't get as far as I wanted creating this.

File: package.json (hunk 1 above)
Comment (abaco) [reply package.json]: Good catch!

**Comment by abaco** [comment]
Cycle now uses xstream internally, so all sources must be `adapt`ed to xstream, as explained in the [migration guide](https://cycle.js.org/releases.html#cycle-unified-for-library-authors-driver-authors). It's a hard requirement of the current Cycle.

I also already ported the all library to xstream - other PR coming soon ;). The advantage is that xstream is much smaller than Rx.

**Review by apoco — APPROVED** [review]

**Comment by apoco** [comment]
Thanks for explaining and for getting this up-to-date.

**Pull request merged**